lib/members_page.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  field :members do
```

Comment:
These aren't really `members`: these are just the URLs. This should either return more information on each one, or be renamed as just `member_urls`.